routing.md

```diff
@@ -277,9 +277,36 @@ Why is such indirection useful?
 * It lets the mediator remain uncommitted to whether the next receiver is another mediator or not. This may provide flexibility in some routing scenarios.
 * It lets the mediator change the size of the message by adding or subtracting noise from the content. 
 * It allows for dynamic routing late in the delivery chain.
- 
+
 These last two characteristics are the foundation of mix networking feature for DIDComm. That feature is the subject of a different RFC; here we only note the existence of the optional feature. 
 
+### DID Document Keys
+
+TODO: include details about how DIDComm keys are represented/identified in the DID Document.
+
+### DID Document Service Endpoint
+
+DIDComm DID Document endpoints have the following format:
+
+```json
+{
+    "id": "did:example:123456789abcdefghi#didcomm-1",
+    "type": "DIDComm",
+    "serviceEndpoint": "http://example.com/path",
+    "routingKeys": ["did:example:somemediator#somekey"]
+}
+```
+
+**id**: must be unique, as required in [DID Core](https://www.w3.org/TR/did-core/#service-endpoints). No special meaning should be inferred from the `id` chosen.
+
+**type**: MUST be `DIDComm`. 
+
+**serviceEndpoint**: MUST contain a URI for a transport specified in the [transports] section of this spec.
+
+**routingKeys**: An optional ordered array of strings referencing keys to be used when preparing the message for transmission as specified in the [Routing] section of this spec. 
```

Comment:
I think we can use the array syntax, and define a JSON-LD context that says that `routingKeys` are an `@list`.... and the desired behavior will be achieved for both representations:

https://json-ld.org/spec/latest/json-ld/#example-51-an-ordered-collection-of-values-in-json-ld